docs/RELEASE.md

```diff
@@ -1,3 +1,50 @@
+# Development Road Map
+The following is a summary of the NNI team's backlog for the next 6 months. Some completed items (either in a release or master branch) are included to provide the context and progress of the work.
```

Comment:
I prefer keep "backlog" as "backlog", which means no "completed" items.

Reply:
Is there any change I should make?